docs/src/main/sphinx/glossary.rst

```diff
@@ -0,0 +1,187 @@
+    In addition to cloud-hosted services, there are also local object storage
+    options such as `MinIO <https://min.io/>`_ and
+    `Ceph <https://docs.ceph.com>`_ that are compatible with S3. Object storage
+    became a popular replacement to :ref:`HDFS <glossHDFS>`.
+
+.. _glossOpenSource:
+
+Open-source
+    Typically refers to
+    `open-source software <https://en.wikipedia.org/wiki/Open-source_software>`_.
+    which is software that has the source code made available for others to see,
+    use, and contribute to. Allowed usage varies depending on the license that
+    the software is licensed under. Trino is licensed under the
+    `Apache license <https://en.wikipedia.org/wiki/Apache_License>`_, and is
+    therefore maintained by a community of contributors from all across the
+    globe.
+
+Privacy-Enhanced Mail (PEM)
+    A syntax for private key information, and a content type used to store and
+    send cryptographic keys and certificates. PEM format can contain both a key
+    and its certificate, plus the chain of certificates from authorities back to
+    the root :ref:`CA <glossCA>`, or back to a CA vendor's intermediate CA.
+
+.. _glossPKCS12:
+
+PKCS #12
+    A binary archive used to store keys and certificates or certificate chains
+    that validate a key. `PKCS #12 <https://en.wikipedia.org/wiki/PKCS_12>`_
+    files have ``.p12`` or ``.pfx`` extensions.
+
+Presto and PrestoSQL
+    The old name for Trino. To learn more about the name change to Trino, read
+    `the history <https://en.wikipedia.org/wiki/Trino_(SQL_query_engine)#History>`_.
+
+Query Federation
+  Query federation .
+  A type of :ref:`data virtualization <glossDataVirtualization>` provides a
+  common access point and data model across two or more heterogeneous data
+  sources. A popular data model used by many query federation engines is
+  translating different data sources to :ref:`SQL <glossSQL>` tables.
+
+.. _glossSQL:
+
+Structured Query Language (SQL)
+    The standard language used with relational databases. For more information,
+    see :doc:`SQL </language>`.
+
+.. _glossSSL:
+
+Secure Socket Layer (SSL)
+    Now superseded by :ref:`TLS <glossTLS>`, but still recognized as the term
+    for what TLS does.
+
+Tarball
+    A common abbreviation for
+    `TAR file <https://en.wikipedia.org/wiki/Tar_(computing)>`_, short for tape
+    archive, which is a common software distribution mechanism. This file is a
+    collection of multiple files distributed as a single file, commonly
+    compressed using :ref:`gzip <glossGzip>` compression.
+
+.. _glossTLS:
+
+Transport Layer Security (TLS)
+    `TLS <https://en.wikipedia.org/wiki/Transport_Layer_Security>`_ is a
+    security protocol designed to provide secure communications over a network.
+    It is used in many applications like HTTPS, email, and even Trino. It is the
+    successor to :ref:`SSL <glossSSL>`. These security topics use the term TLS
```

Comment:
Can condense into one sentence: "It is the successor to SSL, and used in many applications like.."

Reply:
Would remove "even" before "Trino" as unnecessary flair